docs/developing-component.md

```diff
@@ -62,6 +62,7 @@ go mod edit -replace github.com/dapr/components-contrib=../components-contrib
 4. Register your component in dapr [main.go](https://github.com/dapr/dapr/blob/d17e9243b308e830649b0bf3af5f6e84fd543baf/cmd/daprd/main.go#L153-L226)(e.g. binding)
 5. Build debuggable dapr binary
 ```bash
+go mod tidy
```

Comment:
Can you add this to the github workflow too? Similar to how it is done in dapr/dapr, build should fail if `go mod tidy` creates any change.

Reply:
I will have a look

Reply:
Any updates on this?

Reply:
Implemented it based on dapr/dapr. Hope this fits